### PR TITLE
api: fix typo Destroy volume can be recovered

### LINKED ISSUE #5461: volume db update is incorrect
##### ISSUE TYPE

 * Bug Report

##### COMPONENT NAME

~~~
VOLUME
~~~

##### CLOUDSTACK VERSION

~~~
4.15.0.0
4.15.0.1
~~~

##### SUMMARY

This is the VOLUME state definition

Expunging("The volume is being expunging"),
Expunged("The volume has been expunged"),
Destroy("The volume is destroyed, and can't be recovered."),
Destroying("The volume is destroying, and can't be recovered."),

After online reinstall vm, volume has two data in DB, the first one becomes Expunged;
Its status should be "Destroy", right?

`MariaDB [cloud]> select id,pool_id,instance_id,name,uuid,state,removed from volumes where name='ROOT-56';`
`+-----+---------+-------------+---------+--------------------------------------+----------+---------------------+`
`| id  | pool_id | instance_id | name    | uuid                                 | state    | removed             |`
`+-----+---------+-------------+---------+--------------------------------------+----------+---------------------+`
`| 216 |       3 |        NULL | ROOT-56 | 60df5680-23dd-422b-bf7d-d6db2eaa66a0 | Expunged | 2021-09-17 01:13:21 |`
`| 219 |       3 |          56 | ROOT-56 | e28b34c3-7377-4c36-8729-a6715c97e2e1 | Ready    | NULL                |`
`+-----+---------+-------------+---------+--------------------------------------+----------+---------------------+`
`2 rows in set (0.001 sec)`

The following is the change process of this VOLUME event

image

##### STEPS TO REPRODUCE

~~~

~~~

##### EXPECTED RESULTS

~~~

~~~

##### ACTUAL RESULTS

~~~

~~~

### PR DESCRIPTION
**Pull request by weizhouapache**
### Description

This PR fixes #5461 

### Types of changes

- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] Enhancement (improves an existing feature and functionality)
- [ ] Cleanup (Code refactoring and cleanup, that may add test cases)

#### Feature/Enhancement Scale

- [ ] Major
- [ ] Minor

#### Bug Severity

- [ ] BLOCKER
- [ ] Critical
- [ ] Major
- [ ] Minor
- [x] Trivial

### CODE REVIEW

**Review by sureshanaparti — APPROVED** [review]
LGTM

**Review by GabrielBrascher — APPROVED** [review]
LGTM

File: api/src/main/java/com/cloud/storage/Volume.java [hunk 1]
```diff
@@ -48,7 +48,7 @@
         Resizing("The volume is being resized"),
         Expunging("The volume is being expunging"),
         Expunged("The volume has been expunged"),
```
Comment (DaanHoogland) [inline api/src/main/java/com/cloud/storage/Volume.java:50]:
```suggestion
        Expunged("The volume has been expunged, and can no longer be recovered"),
```

**Review by DaanHoogland — APPROVED** [review]
:+1:  maybe change the `Expunged` description as well?

File: api/src/main/java/com/cloud/storage/Volume.java (hunk 1 above)
Comment (weizhouapache) [reply api/src/main/java/com/cloud/storage/Volume.java]: done. thanks @DaanHoogland

**Review by GutoVeronezi — APPROVED** [review]
LGTM

**Comment by sureshanaparti** [comment]
@blueorangutan package

**Review by RodrigoDLopez — APPROVED** [review]
LGTM

**Comment by weizhouapache** [comment]
@sureshanaparti 
it is ready for review

**Comment by sureshanaparti** [comment]
> @sureshanaparti
> it is ready for review

thanks @weizhouapache

**Comment by sureshanaparti** [comment]
state description text change, smoke tests not needed.

**Pull request merged**